source/Reviewers/howtocommit.rst

```diff
@@ -299,24 +299,29 @@ for all affected tests before you commit to the trunk.
         KGO files are stored in `$UMDIR/standard_jobs/kgo` and are installed there
         using a script.
 
-        1. Run the rose stem tasks that require a KGO update, plus any other testing required (see above) - if unsure run the `all` group.
+        1. Run the rose stem tasks that require a KGO update, plus any other testing required (see above) - if unsure run the `all,ex1a` group.
 
             .. code-block::
 
                 rose stem --group=all,ex1a --new
 
-        2. **As yourself**, in your merged Head of Trunk working copy move to ``admin/rose-stem``.
-        3. Run ``meto_update_kgo.sh``. This script will ask you to enter some details regarding the ticket.
+        2. You will need access to both your merged working copy and the SimSys_Scripts github repo. Run the script ``kgo_updates/meto_update_kgo.sh`` which is located in SimSys_Scripts.
```

Comment:
I think it could be clearer that you need a local copy of the simsys_scripts repo, not just access through a github webpage. Perhaps "and a clone of the SimSys_Scripts github repo"?

Reply:
Yep done, and made that a link to the repo